api/src/main/java/org/openmrs/module/kenyaemr/reporting/builder/otz/PatientsOnOTZReportBuilder.java

```diff
@@ -18,6 +18,9 @@
 import org.openmrs.module.kenyacore.report.data.patient.definition.CalculationDataDefinition;
 import org.openmrs.module.kenyaemr.calculation.library.hiv.art.CurrentArtRegimenCalculation;
 import org.openmrs.module.kenyaemr.calculation.library.hiv.art.ViralLoadResultCalculation;
+import org.openmrs.module.kenyaemr.reporting.data.converter.definition.art.ETLLastVisitDateDataDefinition;
```

Comment:
Please move these definitions and their evaluators to the otz package. That will help distinguish them from other similar definitions

Reply:
This one is for the art linelist, not otz